hack/collector-default

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+HOST_PATH=$1
+BUNDLE_DIR=$2
+
+function collect_sysinfo(){
+    cd ${BUNDLE_DIR}
+    mkdir -p hostinfos
+    cd hostinfos
+
+    echo "hostname : " `cat ${HOST_PATH}/etc/hostname` > hostinfo
+    echo "os-release : " `cat ${HOST_PATH}/etc/os-release` >> hostinfo
+    echo "system_info : " `uname -a` >> hostinfo
```

Comment:
how about `system-info`? Make the naming more consistent.

Reply:
Yes, I think it makes item names more consistent. Thanks.
![image](https://user-images.githubusercontent.com/17548901/211155071-1541526c-06ba-4042-8f19-f11d4d38341c.png)